client/ayon_core/pipeline/load/utils.py

```diff
@@ -720,11 +720,13 @@ def get_representation_path(representation, root=None):
         str: fullpath of the representation
 
     """
-
     if root is None:
-        from ayon_core.pipeline import registered_root
+        from ayon_core.pipeline import get_current_project_name, Anatomy
 
-        root = registered_root()
+        anatomy = Anatomy(get_current_project_name())
+        return get_representation_path_with_anatomy(
+            representation, anatomy
+        )
```

Comment:
I can't find any `get_representation_path()` query where we are passing the `root`. Which means that essentially with this change this now always would go through this logic.

Which makes me think, is all the code of "switching between how we're detecting the filepath" below still relevant in any way?

Reply:
Fair point.